src/component/Dialog/FormDialog.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import {observer, inject} from "mobx-react";
+import {Modal, Input, Form} from "antd";
+
+@inject("dialog")
+@inject("content")
+@observer
+class FormDialog extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ...initialState,
+    };
+  }
+
+  buildRow = (rowNum, columnNum) => {
+    let appendText = "|";
+    if (rowNum === 1) {
+      appendText += " --- |";
+      for (let i = 0; i < columnNum - 1; i++) {
+        appendText += " --- |";
+      }
+    } else {
+      appendText += "     |";
+      for (let i = 0; i < columnNum - 1; i++) {
+        appendText += "     |";
+      }
+    }
+    return appendText + (/macintosh|mac os x/i.test(navigator.userAgent) ? "\n" : "\r\n");
+  };
+
+  buildFormFormat = (rowNum, columnNum) => {
+    let formFormat = "";
+    for (let i = 0; i < 3; i++) {
+      formFormat += this.buildRow(i, columnNum);
+    }
+    for (let i = 3; i <= rowNum; i++) {
+      formFormat += this.buildRow(i, columnNum);
+    }
+    return formFormat;
+  };
```

Comment:
we can not konw the final input paramaters value,  so add maxmium limit for rows and columns.

for Ex:
```jsx
buildRow = (rowNum, columnNum) => {
 if(rowNum > 100 || columnNum > 100) {
   // throw a unexpected & handle it
 }
}
```